src/v/compat/metadata_dissemination_compat.h

```diff
@@ -88,4 +88,31 @@ struct compat_check<cluster::update_leadership_request_v2> {
     }
 };
 
+/*
+ * cluster::update_leadership_reply
+ */
+template<>
+struct compat_check<cluster::update_leadership_reply> {
```

Comment:
I wonder if we could have a macro or something for an empty compat_check to save LOC.
```
#define empty_compat_check(class_name)         \
                   template<>                            \
                   struct compat_check<class_name> { \
                   ...
```

Reply:
Nice idea, done.